libs/wingc/src/jsify.rs

```diff
@@ -378,9 +378,8 @@ impl<'a> JSifier<'a> {
 			ExprKind::Reference(_ref) => self.jsify_reference(&_ref, None, context),
 			ExprKind::Call { function, arg_list } => {
 				let function_type = function.evaluated_type.borrow().unwrap();
-				let function_sig = function_type
-					.as_function_sig()
-					.expect("Expected expression to be callable");
+				let function_sig = function_type.as_function_sig();
+				assert!(function_sig.is_some() || function_type.is_anything());
```

Comment:
I think it might be possible to include a message with `assert`
```suggestion
				assert!(function_sig.is_some() || function_type.is_anything(), "Expected expression to be callable");
```